src/bin/wasmer.rs

```diff
@@ -118,6 +120,9 @@ struct Run {
     /// Application arguments
     #[structopt(name = "--", raw(multiple = "true"))]
     args: Vec<String>,
+
+    #[structopt(long = "inst-limit")]
+    instruction_limit: Option<u64>,
```

Comment:
I don't believe this CLI option will be needed and the common use case would be using it as library.